# name       : percona-support.patch
# introduced : 11 or before
# maintainer : Oleg
#
#!!! notice !!!
# Any small change to this file in the main branch
# should be done or reviewed by the maintainer!
diff -ruN a/scripts/mysql_install_db.sh b/scripts/mysql_install_db.sh
--- a/scripts/mysql_install_db.sh       2009-08-08 09:20:07.000000000 +0000
+++ b/scripts/mysql_install_db.sh       2009-08-08 09:29:23.000000000 +0000
@@ -475,6 +475,8 @@
   echo
   echo "Please report any problems with the $scriptdir/mysqlbug script!"
   echo
+  echo "For commercial support please contact Percona at http://www.percona.com/support/"
+  echo
 fi
 
 exit 0
